typescript/infra/config/networks/testnets.ts

```diff
@@ -53,6 +53,26 @@ export const ropsten: TransactionConfig = {
   },
 };
 
+export const bsctestnet: TransactionConfig = {
+  confirmations: 3,
```

Comment:
Are all these confirmation numbers explicitly 3 or just copy-paste?

Reply:
Just copy-paste, I figure bc they're testnet's it's not a big deal and wasn't worth spending the time to get the ideal numbers. wdyt?